rightscript.go

```diff
@@ -835,6 +835,28 @@ func validateRightScript(file string, ignoreMissingMetadata bool) (*RightScript,
 	return &rightScript, nil
 }
 
+func rightScriptCommit(href, message string) {
+	client, _ := Config.Account.Client15()
+
+	// Check if RightScript exists
+	rightscriptLocator := client.RightScriptLocator(href)
+	_, err := rightscriptLocator.Show(rsapi.APIParams{"view": "inputs_2_0"})
```

Comment:
We don't need to make this call. The commit call should give us the same error if it doesn't exist.